packages/venia-ui/lib/components/Checkout/form.css

```diff
@@ -16,6 +16,12 @@
     text-transform: capitalize;
 }
 
+.disabledPrompt {
+    composes: informationPrompt;
+    /* TODO: Is this the best way to signify disabled? Contrast seems light on text. */
+    color: rgb(var(--venia-grey-dark));
```

Comment:
I'd agree maybe we can improve the contrast but since this is consistent with the handful of other "disabled" styles we use in Venia today, I'm 👍 right now.

Reply:
Ok I'll remove the TODO then :)